app_backend/models/user.py

```diff
@@ -13,7 +26,11 @@
         The password of the user
 """
 class User(BaseModel):
-    _id = None
+    id: PyObjectId = Field(default_factory=PyObjectId, alias="_id")
+    class Config:
+        allow_population_by_field_name = True
+        arbitrary_types_allowed = True #required for the _id 
+        json_encoders = {ObjectId: str}
```

Comment:
@DDavid233 you wrote this initially and it was creating a redundant `id` field in the DB. I got rid of it and introduced a bug that seemed to get things by until this feature. I added it back and removed the redundant `id` field by overriding the `dict()` method. Sorry about that!!